src/SourceBuild/tarball/content/README.md

```diff
@@ -0,0 +1,152 @@
+Changes done to the VMR are not automatically mirrored back. More details on this process:
+
+- [Source Synchronization Process](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Design-And-Operation.md#source-synchronization-process)
+- [Moving Code and Dependencies between the VMR and Development Repos](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Design-And-Operation.md#moving-code-and-dependencies-between-the-vmr-and-development-repos)
+
+### Contribution
+
+At this time, the VMR will not accept any changes outside of those necessary for enabling the supported builds to work.
+Please, make the changes in the respective development repositories (e.g., [dotnet/runtime](https://github.com/dotnet/runtime) or [dotnet/sdk](https://github.com/dotnet/sdk)) and they will get synchronized into the VMR automatically.
+
+## Goals
+
+This repository eventually aims to become the place from which we release and service future versions of .NET.
+The goal is to reduce complexity of the product construction process and thus enable partners and 3rd parties to easily build, test and modify .NET using their custom infrastructure.
+
+Furthermore, we hope to solve other problems that the current multi-repo setup brings:
+- Enable developers to make and test changes spanning multiple repositories.
+- Fulfill requirements of .NET distro builders such as RedHat or Canonical to natively include .NET in their distribution repositories.
+- Make it possible for anyone to build the product without the need for currently used CI systems and inter-repo code flow orchestration. This should simplify scenarios such as client-run testing of bug fixes and improvements. The build should work in an offline environment too for certain platforms.
+- Enable the standard [down-/up-stream open-source model](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Upstream-Downstream.md).
+- More efficient pipeline for security fixes during the CVE pre-disclosure process.
+
+## Building the VMR
+
+Please note that **this repository is an experiment and a work-in-progress so it is possible that the build is broken**.
+For the latest information about Source-Build support, please check our [GitHub Discussions page](https://github.com/dotnet/source-build/discussions) for announcements.
+
+### Prerequisites
+
+The dependencies for building .NET from source can be found [here](https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/linux-requirements.md).
+
+### Building
+
+1. Clone the VMR.
+
+   ```bash
+   git clone https://github.com/dotnet/dotnet dotnet-dotnet
+   ```
+
+2. Prep the source to build on your distro. This downloads a .NET SDK and a number of .NET packages needed to build .NET from source.
+
+    ```bash
+    cd dotnet-dotnet
+    ./prep.sh
+    ```
+
+3. Build the .NET SDK
+
+    ```bash
+    ./build.sh --clean-while-building
```

Comment:
Only online builds are supported at the moment.  This instruction should include the `--online` option.

Reply:
Should I call that out in the `Limitations`?